content/css/concepts/grids/grids-grid-template-columns.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: “grids-grid-template-columns”
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - “Grids”
+  - “Layout”
+  - “Template”
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-css"
+  - "https://www.codecademy.com/learn/paths/front-end-engineer-career-path"
+  - "https://www.codecademy.com/learn/paths/full-stack-engineer-career-path"
+---
+
+## Definition
+
+A property that specifies the column structure of a grid container.
+
+## Syntax
+
+```css
+.grid-container {
+	display: grid;
+	grid-template-columns: <column-space> <column-space>;
+}
+```
+
+where `<column-space>` can be one of the following:
+
+- Grid keyword: `auto`, `minmax()`
+- Pixel value: `300px`
+- Percent value: `25%`
+- Fractional unit value: `1fr`
+## Example 1
+
+Split  `spam-container` into four evenly spaced columns:
```

Comment:
Let's describe `spam-container` a bit more:

Split the `spam-container` class into four evenly spaced columns: